config/locales/models/en.yml

```diff
@@ -41,6 +41,11 @@ en:
         name_with_colon:
           one: 'Additional Tag:'
           other: 'Additional Tags:'
+      gift_exchange:
+        offers_num_allowed: Number of offers allowed per sign-up
+        offers_num_required: Number of offers required per sign-up
+        requests_num_allowed: Number of requests allowed per sign-up
+        requests_num_required: Number of requests required per sign-up
```

Comment:
Does having those in the i18n file this allow to clean up a bit view files like [this one](https://github.com/otwcode/otwarchive/blob/8d5b1c97839ec1eb736c3387c1e52e2e30786181/app/views/challenge/gift_exchange/_gift_exchange_form.html.erb#L33)?

Reply:
Hmm. I think it makes migrating those partials to use `I18n.t` a bit easier, but we would still need to include the `:`. So it might be a bit cleaner, but no quite as nice as
```erb
<dt><%= f.label :requests_num_required %></dt>
```
unfortunately. For the purposes of this PR, I put these here to make the field names nicer in error messages

Reply:
You can get rid of the colons in any field labels where you see them, btw -- we actually do not want those.

Reply:
Neat! Another thing I thought of though: the way the allowed/required fields are split, we wouldn't get the correct naming without manually specifying an I18n key

![two rows in a form; the first is labelled Number of requests required per sign-up followed by two text input boxes, labelled Required: and Allowed: respectively; the second is labelled Number of offers required per sign-up followed by two text input boxes, labelled Required: and Allowed: respectively](https://github.com/otwcode/otwarchive/assets/13002992/ec598220-838c-4c88-96ee-e25c97e47a36)